Dockerfile

```diff
@@ -25,20 +30,45 @@ RUN npm install --prefix=$WORKSPACE
 FROM builder as development
 ARG APP
 ENV WORKSPACE apps/${APP}
+# Create app directory
+WORKDIR /opt/openstad-headless
 CMD ["npm", "run", "dev", "--prefix=${WORKSPACE}"]
 
+# Prepare production
+FROM builder as prepare-production
+ARG APP
+ENV WORKSPACE apps/${APP}
+RUN npm --prefix=apps/${APP} prune --production
+
 # Release image
 FROM node:18-slim as release
 ARG APP
 ARG PORT
 ENV WORKSPACE apps/${APP}
 
+LABEL org.opencontainers.image.description="Openstad Headless ${APP} image"
```

Comment:
Had je hier al naar gekeken?

https://docs.docker.com/build/ci/github-actions/manage-tags-labels/
https://github.com/docker/metadata-action?tab=readme-ov-file#overwrite-labels-and-annotations

Reply:
Nee, goede tip! Zojuist ook geprobeerd, en hoewel ik het label correct zie staan in de metadata stap, wordt 'ie niet meegepushed.